eddymotion/estimator.py

```diff
@@ -69,6 +72,8 @@ def fit(
 
         kwargs["S0"] = _advanced_clip(dwdata.bzero)
 
+        kwargs["n_threads"] = n_threads
```

Comment:
```suggestion
        kwargs["n_threads"] = n_threads or cpu_count()
```

Please add `from os import cpu_count` at the top.